cocos/ui/UIButton.cpp

```diff
@@ -788,12 +789,17 @@ void Button::setTitleFontSize(float size)
     {
         _titleRenderer->setSystemFontSize(size);
     }
-    else
+    else if (_type == FontType::TTF)
     {
         TTFConfig config = _titleRenderer->getTTFConfig();
         config.fontSize = size;
         _titleRenderer->setTTFConfig(config);
     }
+    else
+    {
+        // BMFont can't set its font size
+        size = _fontSize;
```

Comment:
is that okay to let the UIButton class to "updateContentSize", and set the member variable "_fontSize" even though nothing change?